File: ОАИП/LR_5-6/2.4.py
arr = list(map(int, input().split()))
max = 0
count = 1
for i in range(1, len(arr)):
    if arr[i-1] <= arr[i]:
        count+=1
        if count > max:
            max = count
    else:
        count = 1
print(max)

File: ОАИП/LR_5-6/2.4.pas
var
  a: array[1..100] of integer;
  i, n, count, max: integer;

begin
  read(n);
  for i := 1 to n do
    read(a[i]);
  max := 0;
  i := 1;
  count := 1;
  for i := 2 to n do
  begin
    if a[i - 1] <= a[i] then
    begin
      count += 1;
    end
    else 
    begin
      if count > max then 
      begin
        max := count
      end;
      count := 1
    end;
  end;
  if count > max then writeln(count)
  else writeln(max)
end.